package.json

```diff
@@ -38,17 +38,17 @@
     "clean": "rm -rf dist/*",
     "dist-standalone": "pkg --compress GZip .",
     "dist-standalone:test": "jest --config ./jest.config-standalone.js",
-    "format": "yarn tslint --fix && yarn prettier --write",
```

Comment:
Could we keep prettier here and in `lint` as some rules are specifically checked by prettier?
(cf. https://github.com/DataDog/datadog-ci/pull/666/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R171)

Reply:
We use a plugin for eslint which runs prettier as part of the `lint` process. (cf. https://github.com/DataDog/datadog-ci/pull/666/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R111).

https://github.com/prettier/eslint-plugin-prettier/blob/master/README.md?plain=1#L79-L91